doc/source/whatsnew/v0.18.1.txt

```diff
@@ -76,8 +76,7 @@ API changes
 
 
 
-
-
+- ``read_csv`` now raises an ``EmptyDataError`` in response to empty columns or header (:issue:`12506`)
```

Comment:
say this is a sub-class of ``ValueError``, which it previously raised, so just a rename change. I would show a full example, before & after to encompass the 'API' changes that are here. IOW, show the previous usage & errors, and the new. (use code-blocks for everything)